### PR TITLE
Add new prometheus-grafana post

### PR DESCRIPTION
**Pull request by nickolamaz**

### CODE REVIEW

File: _posts/2022-06-20-prometheus-grafana.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
+---
+title: "Prometheus Grafana: Configuration & Query Examples"
+description: "Learn the best practices for installing and configuring Grafana to integrate with Prometheus and use PromQL to operators and functions to query its database."
+date: 2022-06-20T10:00:00-06:00
+canonical_url: "https://blog.kubecost.com/blog/prometheus-grafana"
+classes: wide
+categories:
+  - blog
+tags:
+  - Prometheus Grafana
```
Comment (nealormsbee) [inline _posts/2022-06-20-prometheus-grafana.md:10]: Should these be two separate tags?

```
- Prometheus
- Grafana
```

**Review by nealormsbee — APPROVED** [review]
Couple of questions about the metadata but it looks good to me! Merge any time and we can get somebody to check copy and tweak it as needed.

File: _posts/2022-06-20-prometheus-grafana.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
+---
+title: "Prometheus Grafana: Configuration & Query Examples"
+description: "Learn the best practices for installing and configuring Grafana to integrate with Prometheus and use PromQL to operators and functions to query its database."
```
[the remark this answers is not in the document]
Comment (nickolamaz) [reply _posts/2022-06-20-prometheus-grafana.md]: Fixed 👍

File: _posts/2022-06-20-prometheus-grafana.md (hunk 1 above)
Comment (nickolamaz) [reply _posts/2022-06-20-prometheus-grafana.md]: Separated

**Comment by nealormsbee** [comment]
@nickolamaz my mistake! Merging now.

**Pull request merged**